tests/container/Containerfile

```diff
@@ -12,8 +12,19 @@ RUN yum install \
     python3-samba \
     python3-wheel \
     python3-pyxattr \
+    python3-devel \
     python3.9 \
     samba-common-tools \
+    rpm-build \
+    'python3dist(flake8)' \
+    'python3dist(inotify-simple)' \
+    'python3dist(mypy)' \
+    'python3dist(pytest)' \
+    'python3dist(pytest-cov)' \
+    'python3dist(setuptools-scm)' \
+    'python3dist(setuptools-scm)' \
+    'python3dist(tox-current-env)' \
+    'python3dist(wheel)' \
```

Comment:
Are these requirements generated as per `%generate_buildrequires` and/or `%pyproject_buildrequires`?

Reply:
Yes, they're generated automatically by reading the from tox.ini/setup.cfg/etc. I had hoped that using `-e py3` would cause it to leave off certain test-only deps like flake8 and mypy. However, I couldn't figure out how to do that.